Erl.elm

```diff
@@ -322,6 +327,7 @@ parse str =
     hash = (hashFromAll str),
     password = "",
     path = (pathFromAll str),
+    hasTrailingSlash = (hasTrailingSlashFromAll str),
```

Comment:
I'm extracting the path twice here, but I thought I'd ask if that's an optimization you actually wanted before I went making bigger changes.